### PR TITLE
chore: retranslate why GreptimeDB

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

The ZH document has been using the WeChat article, and the content is inconsistent with the EN version.
Now translate the EN version of this document into ZH.

ref to: #1405

## Checklist

- [x] Please confirm that all corresponding versions of the documents have been revised.
- [x] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/concepts/why-greptimedb.md [hunk 1]
```diff
@@ -3,96 +3,112 @@ keywords: [时序数据库, 云原生, 分布式, 高性能, 用户友好, 存
...
-
-然而，在我们的设计中，对 Schema 的定义并非强制性的，而是更偏向于像 MongoDB 这样的数据库的 Schemaless 方式。当数据被写入时，表将被动态地自动创建，新的列（标签和字段）将也会自动被添加进表中。
-
-### PromQL、SQL 和 Python
-
-GreptimeDB 支持 PromQL 和 SQL，它们都依赖于同一查询引擎。该引擎采用向量化执行，这种执行是并行和分布式的。
+## 高性能
 
-PromQL 是可观测领域流行的查询语言，用户可以通过 PromQL 查询汇总由 Prometheus 提供的实时的时序数据。与 SQL 相比，它在使用 Grafana 进行可视化和创建警报规则时要简单得多。GreptimeDB 通过解析 PromQL 转化为查询计划，然后由查询引擎优化和执行，从而原生地、高性能地支持 PromQL。
+在性能优化方面，GreptimeDB 利用 LSM 树、数据分片和基于仲裁的 WAL 设计等不同技术来处理大量的时序数据写入时的工作负载。
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/concepts/why-greptimedb.md:69]: `仲裁` may not be suitable; we can just use English.

```suggestion
在性能优化方面，GreptimeDB 利用 LSM 树、数据分片和基于 Quorum 的 WAL 设计等不同技术来处理大量的时序数据写入时的工作负载。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/concepts/why-greptimedb.md [hunk 2]
```diff
@@ -3,96 +3,112 @@ keywords: [时序数据库, 云原生, 分布式, 高性能, 用户友好, 存
...
+并简化用户的架构、部署和 API。
 
-![The architecture of GreptimeDB](/architecture-2.png)
+阅读 [SQL 示例](/user-guide/overview.md#sql-query-example) 了解详细信息。
 
-接下来，我们将逐一展开说明下这些设计原则。
+## 可用性、可扩展性和弹性
 
-## 统一指标、日志和事件
+从第一天起，GreptimeDB 就按照云原生数据库的原则设计，这意味着它运行并充分利用云的优势。其一些好处包括：
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/concepts/why-greptimedb.md:25]:
```suggestion
从第一天起，GreptimeDB 就按照云原生数据库的原则设计，这意味着它能够充分利用云的优势。其一些好处包括：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/concepts/why-greptimedb.md [hunk 3]
```diff
@@ -3,96 +3,112 @@ keywords: [时序数据库, 云原生, 分布式, 高性能, 用户友好, 存
...
-## 云原生
+![存储/计算分离，计算/计算分离](/storage-compute-disaggregation-compute-compute-separation.png)
 
-GreptimeDB 专为云而生，充分利用云的优势，如弹性、可扩展性和高可用性。
+存储和计算资源是分离的，允许每个资源独立扩展、消耗和定价。
+这不仅大大提高了计算资源的利用率，还允许“按需付费”定价模式，避免了资源未充分利用的浪费。
 
-![Storage/Compute Disaggregation, Compute/Compute separation](/storage-compute-disaggregation-compute-compute-separation.png)
+除了存储和计算隔离，不同的计算资源也被隔离，以避免在数据写入查询、临时查询以及数据压缩或汇总等任务上争用资源，
+从而为大规模实时分析提供新的效率。
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/concepts/why-greptimedb.md:42]: Not very smooth.

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/concepts/why-greptimedb.md [hunk 4]
```diff
@@ -3,96 +3,112 @@ keywords: [时序数据库, 云原生, 分布式, 高性能, 用户友好, 存
...
+### 灵活的架构支持多种部署策略
 
-在云中分离存储和计算资源有几个好处：
+![GreptimeDB 的架构](/architecture-2.png)
 
-- 存储和计算可以根据需求轻松地、独立地进行扩展
-- 数据可以写入价格较低的云存储服务，如阿里云 OSS、 AWS S3 或 Azure Blob Storage
-- 可以使用 Serverless 容器自动且弹性地扩展计算资源
+通过灵活的架构设计原则，不同的模块和组件可以通过模块化和分层设计独立切换、组合或分离。
+例如，我们可以将前端、数据节点和元数据服务模块合并为一个独立的二进制文件，也可以为每个表独立启用或禁用 WAL。
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/concepts/why-greptimedb.md:53]:
```suggestion
例如，我们可以将 Frontend、Datanode 和 Metasrc 模块合并为一个独立的二进制文件，也可以为每个表独立启用或禁用 WAL。
```

**Review by killme2008 — APPROVED** [review]

**Review by sunng87 — APPROVED** [review]

**Pull request merged**